test/jdk/java/awt/TrayIcon/TrayIconPopup/TrayIconPopupTest.java

```diff
@@ -53,7 +54,6 @@
  *          java.desktop/java.awt.Helper
  *          jdk.test.lib.Platform
  *          jtreg.SkippedException
- *          ExtendedRobot
```

Comment:
Since ExtendedRobot is removed /lib/client is no longer required in `@library` tag. Applicable for other tests as well.

Reply:
There are couple of tests with /lib/client in `@library` tag which is no longer required. This cleanup can be handled in subsequent PR.